dev/local-format.js

```diff
@@ -0,0 +1,57 @@
+const fs = require('fs');
+const path = require('path');
+
+const { generateLinks } = require('../.github/workflows/format-release-note');
+
+const getMarkdownFiles = (dir) => {
+  let results = [];
+  const list = fs.readdirSync(dir);
```

Comment:
we can provide the `{ withFileTypes: true }` option to return the files as [fs.Dirent](https://nodejs.org/api/fs.html#class-fsdirent) objects, which have the `.isDirectory()` method on them. that way we don't need to do an extra `statSync()` call

```suggestion
  const list = fs.readdirSync(dir, { withFileTypes: true });
```

Reply:
TIL :D Great tip! Thanks :)